### PR TITLE
Update CITATION.cff

### PR DESCRIPTION
**Pull request by dcherian**
xref #398 

Here was my workflow:

1. Update from Github using [tributors](https://con.github.io/tributors/docs/getting-started)
```
tributors update-lookup github
```

2. Upload CITATION.cff to https://citation-file-format.github.io/cff-initializer-javascript/#/
3. Manually add in new authors :?

Apparently `.zenodo.json` is now redundant so I've dropped it.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -1,43 +1,87 @@
...
+    affiliation: Helmholtz-Zentrum Hereon
+  - family-names: Aoun
+    given-names: Abel
+    orcid: 'https://orcid.org/0000-0003-2289-2890'
+    affiliation: CERFACS
+  - family-names: Davis
+    given-names: Luke
+    orcid: 'https://orcid.org/0000-0003-1072-4638'
+    affiliation: Colorado State University
+  - family-names: Müller
```
Comment (dcherian) [inline CITATION.cff:59]: @MuellerSeb do you have an ORCID you wish to add?

File: CITATION.cff [hunk 2]
```diff
@@ -1,43 +1,87 @@
...
+    given-names: Sebastian
+    affiliation: Helmholtz Centre for Environmental Research
+  - family-names: Vo
+    given-names: Tom
+    orcid: 'https://orcid.org/0000-0002-2461-0191'
+    affiliation: Lawrence Livermore National Laboratory
+  - given-names: Martin
```
Comment (dcherian) [inline CITATION.cff:66]: @sol1105 do you have an affiliation you wish to add?

File: CITATION.cff [hunk 3]
```diff
@@ -1,43 +1,87 @@
...
+    given-names: Jonathan
+    orcid: 'https://orcid.org/0000-0002-5479-0189'
+    affiliation: 'Iowa State University, Ames, IA, USA'
+  - family-names: Magin
+    given-names: Justus
+  - affiliation: 'National Center for Atmospheric Research, USA'
+    family-names: Kent
+    given-names: Julia
+    orcid: 'https://orcid.org/0000-0002-5611-8986'
+  - family-names: Fernandes
```
Comment (dcherian) [inline CITATION.cff:37]: @ocefpaf do you wish to add a ORCID or affiliation

File: CITATION.cff [hunk 4]
```diff
@@ -1,43 +1,88 @@
...
+    orcid: 'https://orcid.org/0000-0003-1192-0403'
+  - affiliation: Axiom Data Science
+    family-names: Thyng
+    given-names: Kristen
+    orcid: 'https://orcid.org/0000-0002-8746-614X'
+  - family-names: Thielen
+    given-names: Jonathan
+    orcid: 'https://orcid.org/0000-0002-5479-0189'
+    affiliation: 'Iowa State University, Ames, IA, USA'
+  - family-names: Magin
```
Comment (dcherian) [inline CITATION.cff:31]: @keewis do you wish to add an affiliation?

File: .tributors [hunk 5]
```diff
@@ -43,6 +45,70 @@
     "withshubh": {
         "name": "Shubhendra Singh Chauhan",
         "bio": "Developer Advocate at @deepsourcelabs  🥑 \r\n👨🏻‍💻 work profile: @shubhendra-deepsource",
-        "blog": "camelcaseguy.com"
+        "blog": "camelcaseguy.com",
+        "orcid": "0000-0001-8475-5571",
+        "affiliation": "Indian Institute of Vegetable Research"
```
Comment (dcherian) [inline .tributors:50]:
```suggestion
        "blog": "camelcaseguy.com",
```

File: CITATION.cff (hunk 3 above)
Comment (ocefpaf) [reply CITATION.cff:38]: Sure,

ORCID: https://orcid.org/0000-0003-4165-2913
Affiliation: IOOS

Should I send a PR or is that OK?

Comment (dcherian) [reply CITATION.cff:38]: I can do it. Thanks!

File: CITATION.cff (hunk 2 above)
Comment (sol1105) [reply CITATION.cff:67]: Yes, thank you

Affiliation: DKRZ (German Climate Computing Centre)

File: .tributors [hunk 6]
```diff
@@ -43,6 +45,68 @@
...
+    "lukelbd": {
+        "name": "Luke Davis",
+        "bio": "PhD candidate in climate science at the Department of Atmospheric Science, Colorado State University.",
+        "orcid": "0000-0003-1072-4638"
+    },
+    "MuellerSeb": {
+        "name": "Sebastian Müller",
+        "bio": "I'm a Model-Keeper at the Department for Computational Hydrosystems within the Helmholtz Centre for Environmental Research (UFZ) in Leipzig.",
+        "blog": "https://geostat-framework.org"
+    },
```
Comment (MuellerSeb) [inline .tributors:104]:
```suggestion
    "MuellerSeb": {
        "name": "Sebastian Müller",
        "orcid": "0000-0001-9060-4008",
        "affiliation": "Helmholtz Centre for Environmental Research - UFZ",
        "bio": "I'm a Model-Keeper at the Department for Computational Hydrosystems within the Helmholtz Centre for Environmental Research (UFZ) in Leipzig.",
        "blog": "https://geostat-framework.org"
    },
```

File: CITATION.cff (hunk 1 above)
Comment (MuellerSeb) [reply CITATION.cff:60]: Thanks for reaching out to me!
```
orcid: 'https://orcid.org/0000-0001-9060-4008'
affiliation: Helmholtz Centre for Environmental Research - UFZ
```

**Review by MuellerSeb — CHANGES_REQUESTED** [review]
Thank you for being so thorough.

**Pull request merged**